.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3-jessie-node-browsers
+    steps:
+      - checkout
+      - run: bundle install
+      - run:
+          name: "Build AsciiDoc files"
+          command: bundle exec asciidoctor src/index.adoc -D dist
+  deploy:
+    docker:
+      - image: circleci/ruby:2.3-jessie-node-browsers
+    steps:
+      - checkout
+      - run: bundle install
+      - run: cp docs.json .docsignore assets -R dist/
+      - run: bundle exec asciidoctor src/index.adoc -D dist
+      - run: sudo npm install -g antman-cli
+      - run: antman upload -a -k $API_KEY https://we.vacuumlabs.com ./dist
```

Comment:
@tomaskulich / @hleumas `API_KEY` should be stored (and _restricted_) in context, currently its in the CircleCI project's ENVs and anyone can just `echo $API_KEY` it out.

- https://circleci.com/docs/2.0/contexts/
- https://circleci.com/docs/2.0/contexts/#restricting-a-context

Reply:
is there anything I can do about this?